docs/eigenda/networks/mainnet.md

```diff
@@ -4,6 +4,20 @@ sidebar_position: 3
 
 # Mainnet
 
-The EigenDA mainnet is coming soon. Stay tuned [on twitter][1] for updates.
+## Quick Links
 
-[1]: https://twitter.com/eigen_da
+* [AVS Page][2]
+* [Blob Explorer][1]
```

Comment:
Maybe add Churner (Dataapi may be left out at the current stage)

Reply:
good call. Added churner address below the disperser address